### ISSUE TITLE
Show where the Original is when used an include Element

### ISSUE THREAD

**Issue by MacKP** [contao/core#7715]
Maybe it is something usefull for the Contao Core -> https://github.com/fritzmg/contao-be-include-info
Without these Infos it is a pain in the a.. ;-)

regards

**Comment by fritzmg** [comment]
True, but it can potentially slow the backend down, since there is at least one additionally database query for each content element of the current article and (in my current implementation) multiple more queries for the actual include elements and the actual element being included. I guess the slowdown is negligible in most cases though.

**Comment by may17** [comment]
This feature is very helpful. Perhaps it`s better to display the information in a modal window?

**Comment by fritzmg** [comment]
True. I am also planning on searching through all text input fields in the database for any possible insert tags - which could actually take a long time. I plan to do this in a modal window, so that the information can be shown on demand. Could be a general solution though.
